packages/frontend/src/stores/messagelist.ts

```diff
@@ -493,14 +493,19 @@ class MessageListStore extends Store<MessageListState> {
 
         this.log.debug('jumpToMessage with messageId: ', jumpToMessageId)
         const accountId = selectedAccountId()
-        // these methods were called in backend before
-        // might be an issue if DeltaBackend.call has a significant delay
 
         if (!accountId) {
           throw new Error('no account set')
         }
 
+        // As was said in this function's docstring,
+        // it should not be called for messages that are in a different chat,
+        // so we know the chatId in advance.
+        // However, let's keep the code that supports arbitrary chatId,
+        // which can be "enabled" by setting `chatIdPreset = undefined`.
+        const chatIdPreset: number | undefined = this.chatId
```

Comment:
You mean that there is (should?) be no use case at the moment, where this.chatId is not set and/or is different from the jumpToMessage.chatId ?
Since later we log an error if isMessageInCurrentChat is false.
If so, you leave the code since it might be useful if we extend the usage later or just as a fallback if this function is called "wrongly" for a message that belongs not to current chat?

Reply:
> You mean that there is (should?) be no use case at the moment, where this.chatId is not set and/or is different from the jumpToMessage.chatId ?

Unless there is a bug that I missed, currently we never call this `jumpToMessage` with a msgId chat belongs to a chat whose ID is not `this.chatId`.

> Since later we log an error if isMessageInCurrentChat is false.

Yes, that would be unexpected.

> If so, you leave the code since it might be useful if we extend the usage later or just as a fallback if this function is called "wrongly" for a message that belongs not to current chat?

I guess it's for the former case mostly.
But maybe also for the latter, because I am not sure if we'd want to just throw an error instead. In this regard I kept things as they were before me.